resting-impl/src/main/java/com/danichef/rest/commands/AdminCommand.java

```diff
@@ -46,10 +46,10 @@ public boolean onCommand(CommandSender sender, Command command, String commandLa
               .forEach(Entity::remove);
 
         } else if ((args.length == 2) && (Bukkit.getPlayer(args[1]) != null)) {
-
             Player target = Bukkit.getPlayer(args[1]);
             if (args[0].equalsIgnoreCase("sit")) {
-                playerSit.sit(target);
+                boolean isSitting = playerSit.sit(target);
+                if (!isSitting) return false;
```

Comment:
Returning `false` in a command generally means "send usage", which is not desired here.
If we want to send a "failed to sit message", that would be okay :+1: